### PR TITLE
[pl] cs-229-machine-learning-tips-and-tricks, cs-229-deep-learning, cs-229-supervised-learning

### PR DESCRIPTION
**Pull request by abecadel**

### CODE REVIEW

**Comment by abecadel** [comment]
Machine learning - done
Deep learning - done

**Comment by shervinea** [comment]
Incredible job, keep up the great work!
In the meantime, we should also wait for another native speaker to review the content before merge.

**Comment by shervinea** [comment]
@abecadel thanks a lot for your hard work!

One last thing before the reviews: a few questions have been added since you created the `pl` folder. Could you please also add to the list the [questions 50-54 of deep learning](https://github.com/shervinea/cheatsheet-translation/blob/master/es/cheatsheet-deep-learning.md), [questions 44-48 of the tips and tricks](https://github.com/shervinea/cheatsheet-translation/blob/master/es/cheatsheet-machine-learning-tips-and-tricks.md) and [questions 88-95 of supervised learning](https://github.com/shervinea/cheatsheet-translation/blob/master/es/cheatsheet-supervised-learning.md)?

**Comment by abecadel** [comment]
@shervinea done

**Comment by shervinea** [comment]
Perfect, thanks @abecadel! Now, let's start the review process.

File: pl/cheatsheet-deep-learning.md [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+
+**26. [Input gate, forget gate, gate, output gate]**
+
+&#10230; [Bramka wejściowa, bramka zapominajca, bramka, bramka wyjściowa]
+
+<br>
+
+**27. [Write to cell or not?, Erase a cell or not?, How much to write to cell?, How much to reveal cell?]**
+
+&#10230; [Pisać do komórki, czy nie?, Wyczyścić komówke, czy nie?, Jak dużo zapisać do komórki?, Jak dużo ujawnić komórce?]
```
Comment (mbiesiad) [inline pl/cheatsheet-deep-learning.md:159]: 159 should be without redundant punctuation marks, eg. (...) czy nie? Wyczyścić (...), not "czy nie?, Wyczyścić" ["," after "?"] etc.

File: pl/cheatsheet-deep-learning.md [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+
+**15. Step 1: Take a batch of training data.**
+
+&#10230; Krok 1: Pobierz pakiet danych treningowych.
+
+<br>
+
+**16. Step 2: Perform forward propagation to obtain the corresponding loss.**
+
+&#10230; Krok 2: dokonaj propagacji do przodu aby uzyskać wartość straty.
```
Comment (mbiesiad) [inline pl/cheatsheet-deep-learning.md:93]: small typo -> "," before "aby"

File: pl/cheatsheet-deep-learning.md [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+
+**17. Step 3: Backpropagate the loss to get the gradients.**
+
+&#10230; Step 3: Z wykorzystaniem propagacji wstecznej użyj straty aby uzyskać gradient.
+
+<br>
+
+**18. Step 4: Use the gradients to update the weights of the network.**
+
+&#10230; Krok 4: Wykorzystaj gradient aby zaktualizować wagi w sieci neuronowej.
```
Comment (mbiesiad) [inline pl/cheatsheet-deep-learning.md:105]: the same

File: pl/cheatsheet-deep-learning.md [hunk 4]
```diff
@@ -0,0 +1,321 @@
...
+
+**45. 2) We iterate the value based on the values before:**
+
+&#10230; W iteracyjny sposób ustalamy wartość zmiennej w oparciu o wartość poprzedniej zmiennej:
+
+<br>
+
+**46. Maximum likelihood estimate ― The maximum likelihood estimates for the state transition probabilities are as follows:**
+
+&#10230; Szacowanie maksymalnego prawdopodobieństwa - Szacowanie maksymalnego prawdopodobieństwo dla poszczególnych przejść pomiędzy stanami wygląda następująco:
```
Comment (mbiesiad) [inline pl/cheatsheet-deep-learning.md:273]: small typo - "Szacowanie maksymalnego prawdopodobieństwo" should be: Szacowanie maksymalnego prawdopodobieństwa

File: pl/cheatsheet-deep-learning.md [hunk 5]
```diff
@@ -0,0 +1,321 @@
...
+
+**53. [Recurrent Neural Networks, Gates, LSTM]**
+
+&#10230; [Rekurencyjne Sieci Neuronowe, Bramki, LSTM]
+
+<br>
+
+**54. [Reinforcement learning, Markov decision processes, Value/policy iteration, Approximate dynamic programming, Policy search]**
+
+&#10230; [Uczenie wspoagane, Proces decyzyjny Markowa, Iteracja wartość/strategia, Przybliżone programowanie dynamiczne, Wyszukiwanie strategii]
```
Comment (mbiesiad) [inline pl/cheatsheet-deep-learning.md:321]: small typo -> should be: Uczenie wspomagane

**Review by mbiesiad — COMMENTED** [review]
few typos

**Pull request closed without merging**